articles/windows-mdm-setup.md

```diff
@@ -141,6 +143,37 @@ Testing automatic enrollment requires creating a test user in Microsoft Entra ID
 2. After it's been wiped, open your workstation and follow the setup steps. At screen in which you're asked to sign in, you should see the title "Welcome to [your organziation]!" next to the logo you uploaded in step 4.
 
 
+## Automatic Windows MDM Migration
+
+Fleet can automatically migrate your Windows hosts from a 3rd party MDM solution to Fleet MDM. Once configured, the migration happens quickly and doesn't require any end user interaction.
+
+### Step 1: set up Windows MDM in Fleet
+
+Follow the [steps above](#manual-enrollment) to turn on Windows MDM in Fleet. 
+
+### Step 2: install Fleet's agent on the hosts
+
+1. Make sure to install [Fleet's agent (fleetd)](https://fleetdm.com/docs/using-fleet/enroll-hosts) on the Windows hosts you want to migrate from the 3rd party MDM solution.
+
+2. Navigate to the **Hosts** tab in the main navigation bar and wait until your hosts are visible in the hosts list.
+
+### Step 3: enable automatic migration
+
+1. Head back to the **Settings > Integrations > Mobile device management (MDM)** page.
+
+2. Next to **Windows MDM turned on (servers excluded)** select **Edit** to navigate to the **Manage Windows MDM** page.
+
+3. On the **Manage Windows MDM** page, select **Automatically migrate hosts connected to another MDM solution**. Click **Save** to save the change.
+
+### Step 4: monitor your hosts as they migrate to Fleet MDM
+
+Once the automatic migration is enabled, Fleet will send a notification to the agent on each host to tell it to migrate from the 3rd party MDM solution to Fleet MDM. This process usually takes a few minutes at most.
```

Comment:
```suggestion
Once the automatic migration is enabled, Fleet sends a notification to each host to tell it to migrate. This process usually takes a few minutes at most.
```

I think we can simplify language here. Will be easier to maintain going forward.